app/internal/page/how_would_certificate_provider_prefer_to_carry_out_their_role.go

```diff
@@ -0,0 +1,90 @@
+package page
+
+import (
+	"fmt"
+	"net/http"
+	"net/mail"
+
+	"github.com/ministryofjustice/opg-go-common/template"
+)
+
+type howWouldCertificateProviderPreferToCarryOutTheirRoleData struct {
+	App                 AppData
+	Errors              map[string]string
+	CertificateProvider CertificateProvider
+	Form                *howWouldCertificateProviderPreferToCarryOutTheirRoleForm
+}
+
+func HowWouldCertificateProviderPreferToCarryOutTheirRole(tmpl template.Template, lpaStore LpaStore) Handler {
+	return func(appData AppData, w http.ResponseWriter, r *http.Request) error {
+		lpa, err := lpaStore.Get(r.Context(), appData.SessionID)
+		if err != nil {
+			return err
+		}
+
+		if len(lpa.PeopleToNotify) > 0 {
+			return appData.Lang.Redirect(w, r, lpa, Paths.ChoosePeopleToNotifySummary)
+		}
```

Comment:
Why are we interested in PeopleToNotify here?

Reply:
ah the ol copy pasta